e2e/suite1/specs/query-editor.spec.ts

```diff
@@ -29,7 +29,7 @@ e2e.scenario({
 
     cy.contains(queryText.slice(0, -1)).should('be.visible');
 
-    e2e.components.QueryField.container().type('{ctrl}z');
+    e2e.components.QueryField.container().type('{cmd}z');
```

Comment:
as noted in the description, this is unrelated. at the ready to revert if it breaks on CI or something